pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -309,13 +336,27 @@ class GammaGammaModelIndividual(BaseGammaGammaModel):
         .. code-block:: python
 
             import pymc as pm
-            from pymc_marketing.clv import GammaGammaModel
-
-            model = GammaGammaModel(
-                customer_id=[0, 0, 0, 1, 1, 2, ...],
-                individual_transaction_value=[5.3. 5.7, 6.9, 13.5, 0.3, 19.2 ...],
+            from pymc_marketing.clv import GammaGammaModelIndividual
+
+            model = GammaGammaModelIndividual(
+                data=pd.DataFrame({
+                    "customer_id": [0, 0, 0, 1, 1, 2, ...],
+                    "individual_transaction_value": [5.3. 5.7, 6.9, 13.5, 0.3, 19.2 ...],
+                }),
+                model_config={
+                    "p_prior": {dist: 'HalfNorm', kwargs: {}},
+                    "q_prior": {dist: 'HalfStudentT', kwargs: {}},
+                    "v_prior": {dist: 'HalfCauchy', kwargs: {}},
```

Comment:
Same issue

Reply:
I've replaced it empty kwargs dict with {"nu": 4, "sigma": 10} as in other examples